client/src/pages/Scores/Scores.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react'
+import { RouteComponentProps } from '@reach/router'
+import { Game } from '../../types/Game'
+import './Scores.scss'
+import Button from '../../components/Button'
+import { GameStatus } from '../../types/GameStatus'
+import { RoundResult } from '../../types/RoundResult'
+import { Player } from '../../types/Player'
+import ScoreBoard from '../../components/ScoreBoard'
+
+export interface ScoresProps extends RouteComponentProps {
+  game: Game
+  currentPlayerId: string
+  isGameHost: boolean
+  result?: RoundResult | undefined
+  startGame: () => void
+}
+
+const Scores: FC<ScoresProps> = (props: ScoresProps) => {
+  const buildResultText = (
+    result: RoundResult,
+    players: Array<Player>
+  ): string => {
+    if (!result.playerId) return `Personne n'a tapé sur la pile`
+    else {
+      let firstPart = ''
+      if (result.playerId === props.currentPlayerId) {
+        firstPart = 'Vous avez'
+      } else {
+        const player = players.find(player => player.id === result.playerId)
+        firstPart = `${(player && player.name) || 'Un joueur inconnu'} a`
+      }
+      return `${firstPart} ${result.hasWon ? 'gagné' : 'perdu'}, `
+    }
+  }
+
+  return (
+    <div className='Scores'>
+      {props.result && (
```

Comment:
Might be unecessary if `props.result` can not actually be `undefined`

Reply:
✅